CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/TeamsAPI/Models/ComposeExtensionSuggestedAction.cs

```diff
@@ -0,0 +1,37 @@
+// <auto-generated /> Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Bot.Connector.Teams.Models
+{
+    using System.Linq;
+
+    /// <summary>
+    /// Compose extension Actions (Only when type is auth or config)
+    /// </summary>
+    public partial class ComposeExtensionSuggestedAction
```

Comment:
@maoyu-dev This name is singular (XXXAction) but it actually contains a list of actions (plural). Can we change it to ComposeExtensionSuggestedActions?